src/main/java/in/projecteka/consentmanager/consent/model/ConsentArtefact.java

```diff
@@ -14,6 +14,9 @@
 @AllArgsConstructor
 @NoArgsConstructor
 public class ConsentArtefact implements Serializable {
+    public static final String CA_SCHEMA_VERSION = "v1.0.0";
+    @Builder.Default
+    private String schemaVersion = CA_SCHEMA_VERSION;
```

Comment:
Can you please persist the schema version also? 

The thing about version to actual schema can be figured out later, when we are going to deal with one API version handling multiple schema versions, until then it's okay to have one to one mapping of URL versus schema.

If you are okay, you need to update the swagger contract in CM.

And also you need to modify the HIU and HIP consent artefact too hold the schemaVersion.

Reply:
Is there a suggested place / constants or something of that sort?

Reply:
What I'm thinking right now, we will have separate branch for each version of the API. If you agree, we can keep it in the constants.

You could create a constant class under, https://github.com/ProjectEKA/consent-manager/blob/master/src/main/java/in/projecteka/consentmanager/common (Use generic variable name for example API_VERSION/VERSION), as we have structured our consent-manager in such a way that, it can be migrated to different deployments in case needed.

Reply:
Agree. lets keep it external, we can discuss and if required refactor for multiple version support.